### ISSUE TITLE
a11y

### ISSUE THREAD

**Issue by RichDom2185** [source-academy/frontend#2613]
Accessibility should be something we strive for, especially being an online learning environment/platform.

Let us gradually adopt accessible components and pages across the entire application.
